### ISSUE TITLE
Hellweg:  Win10/Chrome:  drag-and-drop of elements on Lattice tab

### ISSUE THREAD

**Issue by bruhwiler** [radiasoft/sirepo#869]
Drag-and-drop on the Hellweg lattice page doesn't work on Windows 10 with the latest version of the Chrome browser.

It works fine with Microsoft Edge and also with Firefox.

**Comment by moellep** [comment]
I'm not able to reproduce this with my Windows 10/Chrome laptop. Is there a specific simulation which causes the problem?

**Comment by bruhwiler** [comment]
I see the problem with all the examples and with newly created simulations.
Here's one example, https://beta.sirepo.com/hellweg#/lattice/iwLWvhut

**Comment by robnagler** [comment]
Can you capture a movie?

**Comment by bruhwiler** [comment]
I started making a movie, using screencastify, to demonstrate the problem.
During the course of the movie, drag and drop started to work correctly.
Very weird.

I stopped the movie and went through the examples, and now the problem is resolved.

I'm guessing my browser was in an unusual state...

In any case, it seems that the problem was intermittent. I apologize for the waste of time. I'll close this issue.
